test/ssl-tests/18-dtls-renegotiate.cnf

```diff
@@ -1,294 +1,4 @@
-Method = DTLS
-ResumptionExpected = No
-UseSCTP = No
-client = 5-renegotiate-aead-to-non-aead-client-extra
-
-[5-renegotiate-aead-to-non-aead-client-extra]
-RenegotiateCiphers = AES128-SHA
-
-
-# ===========================================================
-
-[6-renegotiate-non-aead-to-aead]
-ssl_conf = 6-renegotiate-non-aead-to-aead-ssl
-
-[6-renegotiate-non-aead-to-aead-ssl]
-server = 6-renegotiate-non-aead-to-aead-server
-client = 6-renegotiate-non-aead-to-aead-client
-
-[6-renegotiate-non-aead-to-aead-server]
-Certificate = ${ENV::TEST_CERTS_DIR}/servercert.pem
-CipherString = DEFAULT
-Options = NoResumptionOnRenegotiation
-PrivateKey = ${ENV::TEST_CERTS_DIR}/serverkey.pem
-
-[6-renegotiate-non-aead-to-aead-client]
-CipherString = AES128-SHA
-MaxProtocol = DTLSv1.2
-VerifyCAFile = ${ENV::TEST_CERTS_DIR}/rootcert.pem
-VerifyMode = Peer
-
-[test-6]
-ExpectedResult = Success
-HandshakeMode = RenegotiateClient
-Method = DTLS
-ResumptionExpected = No
-UseSCTP = No
-client = 6-renegotiate-non-aead-to-aead-client-extra
-
-[6-renegotiate-non-aead-to-aead-client-extra]
-RenegotiateCiphers = AES128-GCM-SHA256
-
-
-# ===========================================================
-
-[7-renegotiate-non-aead-to-non-aead]
-ssl_conf = 7-renegotiate-non-aead-to-non-aead-ssl
-
-[7-renegotiate-non-aead-to-non-aead-ssl]
-server = 7-renegotiate-non-aead-to-non-aead-server
-client = 7-renegotiate-non-aead-to-non-aead-client
-
-[7-renegotiate-non-aead-to-non-aead-server]
-Certificate = ${ENV::TEST_CERTS_DIR}/servercert.pem
-CipherString = DEFAULT
-Options = NoResumptionOnRenegotiation
-PrivateKey = ${ENV::TEST_CERTS_DIR}/serverkey.pem
-
-[7-renegotiate-non-aead-to-non-aead-client]
-CipherString = AES128-SHA
-MaxProtocol = DTLSv1.2
-VerifyCAFile = ${ENV::TEST_CERTS_DIR}/rootcert.pem
-VerifyMode = Peer
-
-[test-7]
-ExpectedResult = Success
-HandshakeMode = RenegotiateClient
-Method = DTLS
-ResumptionExpected = No
-UseSCTP = No
-client = 7-renegotiate-non-aead-to-non-aead-client-extra
-
-[7-renegotiate-non-aead-to-non-aead-client-extra]
-RenegotiateCiphers = AES256-SHA
-
-
-# ===========================================================
-
-[8-renegotiate-aead-to-aead]
-ssl_conf = 8-renegotiate-aead-to-aead-ssl
-
-[8-renegotiate-aead-to-aead-ssl]
-server = 8-renegotiate-aead-to-aead-server
-client = 8-renegotiate-aead-to-aead-client
-
-[8-renegotiate-aead-to-aead-server]
-Certificate = ${ENV::TEST_CERTS_DIR}/servercert.pem
-CipherString = DEFAULT
-Options = NoResumptionOnRenegotiation
-PrivateKey = ${ENV::TEST_CERTS_DIR}/serverkey.pem
-
-[8-renegotiate-aead-to-aead-client]
-CipherString = AES128-GCM-SHA256
-MaxProtocol = DTLSv1.2
-VerifyCAFile = ${ENV::TEST_CERTS_DIR}/rootcert.pem
-VerifyMode = Peer
-
-[test-8]
-ExpectedResult = Success
-HandshakeMode = RenegotiateClient
-Method = DTLS
-ResumptionExpected = No
-UseSCTP = No
-client = 8-renegotiate-aead-to-aead-client-extra
-
-[8-renegotiate-aead-to-aead-client-extra]
-RenegotiateCiphers = AES256-GCM-SHA384
-
+num_tests = 0
```

Comment:
This looks like a bug

Reply:
Yes, I tink you want to reinstate this file and only remove the UseSCTP bits.

Reply:
It's supposed to be generated by the cnf.dat, so a bug there made this vanish. (This is annoying that both are checked in...)

Reply:
Oh I see. I was wondering how this happened... The never ending joys of complexity in this code base.

Reply:
Turns out i made a typo in the regen command. That's fixed. I did completely reindent the source file though: just too far from what the editor wants to do in their layout.